### ISSUE TITLE
SyntaxError: Unexpected token function

### ISSUE THREAD

**Issue by towry** [parcel-bundler/parcel#277]
## bug report:

```bash
/Users/towry/.nvm/versions/node/v6.9.1/lib/node_modules/parcel-bundler/bin/cli.js:84
async function bundle(main, command) {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
```

### 🌍 Your Environment

| Software         | Version(s)
| ---------------- | ----------
| Parcel           |  NaN
| Node             | v6.9.1
| npm/Yarn         | npm or Yarn
| Operating System | macOS 10.13.2

**Comment by towry** [comment]
Under node v8.9.3 it works, but here is another exception message printed out.

```bash
➜  ~ parcel
⏳  Building...
Server running at http://localhost:1234
🚨  Cannot find module '/Users/towry' from '.'
    at /Users/towry/.nvm/versions/node/v8.9.3/lib/node_modules/parcel-bundler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/Users/towry/.nvm/versions/node/v8.9.3/lib/node_modules/parcel-bundler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/towry/.nvm/versions/node/v8.9.3/lib/node_modules/parcel-bundler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/towry/.nvm/versions/node/v8.9.3/lib/node_modules/parcel-bundler/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:152:21)
```

**Comment by DeMoorJasper** [comment]
Should be fixed once this gets merged #246
Also this is a duplicate of #244

**Comment by towry** [comment]
Cant wait for it.
